src/azure/cli/commands/arm.py

```diff
@@ -226,6 +252,11 @@ def __call__(self, parser, namespace, values, option_string=None):
     cmd = CliCommand(name, handler, simple_output_query=simple_output_query)
     cmd.arguments.update(set_arguments)
     cmd.arguments.update(get_arguments)
+    if function_arguments:
+        cmd.arguments.update(function_arguments)
+    cmd.arguments.pop('instance', None) # inherited from custom_function(instance, ...)
+    cmd.arguments.pop('parent', None)
+    cmd.arguments.pop('expand', None) # possibly inherited from the getter
```

Comment:
>cmd.arguments.pop('expand', None) # possibly inherited from the getter [](start = 4, length = 70)

I'm hesitant to pop an arg we don't own, it seems like this is also a corner case (not many commands using this).  Can we just hide this on the individual commands that have them?